downloads/instructions/macOS_22.02.beta.02.md

```diff
@@ -51,27 +51,19 @@ Upon opening the OpenRocket application for the first time, you can get some sec
     </tr>
     <tr>
       <td>
-        <img src="downloads/instructions/img/macOS_22.02.beta.02/02.security_1.png" alt="Mounted installation disk image" width="240">
+        <img src="downloads/instructions/img/macOS_22.02.beta.02/02.finder.png" alt="OpenRocket beta icon in finder window with options menu showing" width="240">
       </td>
       <td>
-       If you get this pop-up, <strong>double-click the "OK" button</strong> in the security dialog.
+       To open OpenRocket the first time <strong>right-click, command-click, or two finger click the application icon</strong> in finder to get it's options menu, <strong>select "Open"</strong> from the menu.
```

Comment:
Maybe start with mentioning that the user should locate the OpenRock app in the Applications-folder?

Also small linguistic fixes: write "finder" with a capital: "Finder" + change "it's" to "its"

I would suggest:
```
Locate the OpenRocket application in Finder in the Application-directory. To open OpenRocket <strong>right-click, control-click, or two finger click the application icon</strong> in Finder to get its options menu and then <strong>select "Open"</strong> from the menu. Or <strong>double-click the application icon</strong>.
```

Reply:
double clicking the application icon will NOT present the security option to open the application, it has to be done via a "non-default" action. That's part of Apple's security mechanism. So that part of your suggestion I'm going to drop, the rest of it is great... additional commit shortly.

Reply:
That's something I didn't know, so it's best to explicitly mention to not double-click it the first time, since that's something I do all the time.